### ISSUE TITLE
Labels, labels , labels

### ISSUE THREAD

**Issue by delvedor** [fastify/fastify#1521]
Hello team!
As you probably have imagined, we heavily use labels in our issues and prs, not only because they are useful for searching for issues and let people know what to expect from a issue, but also because it makes way easier for me or @mcollina create a new release.

Since we are supporting two versions of the framework at the same time, and it the future it may be even 3, I would like to propose new labels.
`1.x` for every issue/pr related to version `1.x` and so on (and always use the same color).
Then if we make a pr for `2.x` that should be ported to `1.x`, that pr will have at least two labels, `2.x` and `backport-to-1.x` (I'm thinking about a better name for this one).

What do you think?

Finally, a request, please remember always to use labels when creating a pr, it makes the release process way easier :)
Thanks!

**Comment by jsumners** [comment]
👍

**Comment by mcollina** [comment]
I think we should have a “backport-requested-1.x” label, which is removed once the backport is done. Wdyt?

**Comment by delvedor** [comment]
> I think we should have a “backport-requested-1.x” label, which is removed once the backport is done. Wdyt?

I like the idea, but labels with long names are more difficult to read, probably `backport-1.x` is enough.

**Comment by Eomm** [comment]
👍🏼 For the labels

Could we add also a little description for all the labels? 
Because "missing discussion" and "wontfix" are not clear for me 😅 (because there is the discussion and invalid labels that are quite similar to me)

**Comment by delvedor** [comment]
> Could we add also a little description for all the labels?

👍

**Comment by delvedor** [comment]
Oks, I've updated the labels, let me know what you think.
https://github.com/fastify/fastify/labels

I've also added a brief description to the not obvious ones.
